### PR TITLE
Emit structured JSONL log messages

### PR DESCRIPTION
**Pull request by antifuchs**
Previously, we logged colorized messages, which isn't great for log aggregators and queryability in general. Let's use JSONL.

### CODE REVIEW

File: eip_operator_shared/src/lib.rs [hunk 1]
```diff
@@ -259,15 +259,19 @@ where
...
+                .with_filter(MyEnvFilter(EnvFilter::from_default_env()));
             tracing_subscriber::Registry::default()
                 .with(otel_layer)
                 .with(stdout_layer)
                 .init();
         }
         Err(_) => {
-            tracing_subscriber::fmt::init();
+            tracing_subscriber::fmt()
+                .with_env_filter(EnvFilter::from_default_env())
```
Comment (guswynn) [inline eip_operator_shared/src/lib.rs:272]: oh we were missing this before, nice catch

**Review by guswynn — APPROVED** [review]
nice

File: eip_operator_shared/src/lib.rs (hunk 1 above)
Comment (antifuchs) [reply eip_operator_shared/src/lib.rs:272]: Hah, that's equivalent to `::init()`, according to its docs - but the new one allows us to do `.json()` (:

Comment (guswynn) [reply eip_operator_shared/src/lib.rs:272]: ugh, `::init()` defaults :(

**Pull request merged**